subprojects/enterprise-operations/src/main/java/org/gradle/operations/execution/CachingDisabledReasonCategory.java

```diff
@@ -36,25 +35,32 @@ public enum TaskOutputCachingDisabledReasonCategory {
      */
     NOT_ENABLED_FOR_TASK,
 
+    /**
+     * Caching has not been enabled for the unit of work.
+     *
+     * @since 8.3
+     */
+    NOT_CACHEABLE,
+
     /**
      * The task has no outputs declared.
      */
     NO_OUTPUTS_DECLARED,
 
     /**
-     * Task has a {@link org.gradle.api.file.FileTree} or {@link org.gradle.api.internal.file.collections.DirectoryFileTree} as an output.
+     * Task has a {@code org.gradle.api.file.FileTree} or {@code org.gradle.api.internal.file.collections.DirectoryFileTree} as an output.
      *
      * @since 5.0
      */
     NON_CACHEABLE_TREE_OUTPUT,
 
     /**
-     * Caching is disabled for the task via {@link org.gradle.api.tasks.TaskOutputs#cacheIf(Spec)}.
+     * Caching is disabled for the task via {@code org.gradle.api.tasks.TaskOutputs#cacheIf(Spec)}.
      */
     CACHE_IF_SPEC_NOT_SATISFIED,
 
     /**
-     * Caching is disabled for the task via {@link org.gradle.api.tasks.TaskOutputs#doNotCacheIf(String, Spec)}.
+     * Caching is disabled for the task via {@code org.gradle.api.tasks.TaskOutputs#doNotCacheIf(String, Spec)}.
```

Comment:
🤔 These Javadocs talk about tasks, but instead they should be talking about work. Mentioning task-specific APIs is also strange.

Reply:
Good catch, I adjusted the Javadoc.